content/en/docs/examples/virtual-machines/single-network/index.md

```diff
@@ -140,60 +158,71 @@ following commands on a machine with cluster admin privileges:
     As an example, for very simple demo setups, you can also use:
 
     {{< text bash >}}
-    $ go run istio.io/istio/security/tools/generate_cert \
-          -client -host spiffee://cluster.local/vm/vmname --out-priv key.pem --out-cert cert-chain.pem  -mode citadel
-    $ kubectl -n istio-system get cm istio-ca-root-cert -o jsonpath='{.data.root-cert\.pem}' > root-cert.pem
+        $ kubectl -n "${SERVICE_NAMESPACE}" get configmaps istio-ca-root-cert -o json | jq -j '."data"."root-cert.pem"' > root-cert
     {{< /text >}}
 
 ### Setting up the VM
 
 Next, run the following commands on each machine that you want to add to the mesh:
 
-1.  Copy the previously created `cluster.env` and `*.pem` files to the VM. For example:
+1.  Copy the previously created `cluster.env`,`root-cert`,`istio-token`, and `sidecar.env` to the VM. For example:
 
     {{< text bash >}}
     $ export GCE_NAME="your-gce-instance"
-    $ gcloud compute scp --project=${MY_PROJECT} --zone=${MY_ZONE} {key.pem,cert-chain.pem,cluster.env,root-cert.pem} ${GCE_NAME}:~
+    $ gcloud compute scp --project=${MY_PROJECT} --zone=${MY_ZONE} {cluster.env,root-cert,sidecar.env,istio-token} ${GCE_NAME}:~
```

Comment:
no platform specific instructions in the documentaiton please. instead - recommend ssh to copy the files.